test/LendingPool.t.sol

```diff
@@ -1494,3 +1508,211 @@ contract VaultTest is LendingPoolTest {
         assertEq(amountLoaned, openPosition);
     }
 }
+
+/* //////////////////////////////////////////////////////////////
```

Comment:
Add the new tests under the corresponding tests of the functions.
Up till now the functions have the same order as in their corresponding solidity file

Reply:
unit tests for the function depositInLendingPool() should be together with all other unit-tests for depositInLendingPool(), otherwise it is very hard to check that we cover all happy and unhappy flows per test

Reply:
Added tests can be seen in: https://github.com/arcadia-finance/arcadia-lending/pull/37/commits/04d2cf10f649c85ce714ecbab34e25b5832fb362. These what you meant?

Reply:
i don't see added value of duplicating tests other than more refactor work

Reply:
> Added tests can be seen in: [04d2cf1](https://github.com/arcadia-finance/arcadia-lending/commit/04d2cf10f649c85ce714ecbab34e25b5832fb362). These what you meant?

Yes